src/main/java/gdx/liftoff/ui/panels/CompleteButtonsPanel.java

```diff
@@ -68,23 +72,37 @@ public void populate(boolean fullscreen) {
         table.add(ideaButton);
         addHandListener(ideaButton);
         try {
-            List<String> findIntellijExecutable = (UIUtils.isWindows) ? Arrays.asList("where.exe", "idea") : Arrays.asList("which", "idea");
+            if (UIUtils.isWindows) {
+                List<String> findIntellijExecutable = Arrays.asList("where.exe", "idea");
 
-            Process whereProcess = new ProcessBuilder(findIntellijExecutable).start();
-            if (whereProcess.waitFor() == 0) {
-                intellijPath = new BufferedReader(new InputStreamReader(whereProcess.getInputStream())).readLine();
-                ideaButton.setDisabled(false);
-            } else {
-                if (UIUtils.isLinux) {
-                    intellijPath = findFlatpakIntellij();
-                    intellijIsFlatpak = true;
+                Process whereProcess = new ProcessBuilder(findIntellijExecutable).start();
+
+                if (whereProcess.waitFor() == 0) {
+                    intellijPath = new BufferedReader(new InputStreamReader(whereProcess.getInputStream())).readLine();
                     ideaButton.setDisabled(false);
-                } else if (UIUtils.isWindows) {
+                } else {
                     intellijPath = findManuallyInstalledIntellijOnWindows();
                     ideaButton.setDisabled(false);
+                }
+            } else if (UIUtils.isLinux) {
+                List<String> whichCommand = new ArrayList<>();
+                whichCommand.add("which");
+                whichCommand.addAll(manuallyInstalledLinuxIdeaNames);
+
+                Process whichProcess = new ProcessBuilder(whichCommand).start();
+
+                int exitCode = whichProcess.waitFor();
```

Comment:
In most places, final doesn't do much. If it's exposed to outside the function, then it can matter quite a bit because it affects how the code can be used, but it generally doesn't affect performance to mark a local variable as final. Since at least as old as Java 8, HotSpot has been able to infer finality on locals that are "effectively final" because they are required to be final or effectively final to be used in lambdas. The compiler can optimize effectively final variables in the same ways as final ones, I believe under the same assumption that reflection isn't tinkering with the bytecode at runtime.